tests/testStreamOpen.cpp

```diff
@@ -20,6 +20,15 @@
 
 using namespace oboe;
 
+class NumFramesCallback : public AudioStreamCallback {
+public:
+    DataCallbackResult onAudioReady(AudioStream *oboeStream, void *audioData, int32_t numFrames) override {
+		framesPerCallback = numFrames;
+        return DataCallbackResult::Continue;
+    }
+    
+    std::atomic<int32_t> framesPerCallback{0};
```

Comment:
Maybe add a comment saying `// This is exposed publicly so that the number of frames per callback can be tested`.